pipelines/kubeflow_pipelines_component_examples/filter_file_component/README.md

```diff
@@ -3,8 +3,7 @@
 Use the [_filter text_ example component](https://github.com/elyra-ai/elyra/blob/master/etc/config/components/kfp/filter_text_using_shell_and_grep.yaml) to find lines in a text file that contain the specified string pattern.
 
 ### Prerequisites
-
-None. The example pipeline should work as is.
+To use this component, ensure that you create your pipeline with the accompanying `textfile` in your Jupyterlab workspace root directory.
 
```

Comment:
To review and run the example pipeline for this component, [clone the examples repository](https://github.com/elyra-ai/examples) and launch JupyterLab as shown below:

```
jupyter lab --NotebookApp.notebook_dir=examples/pipelines/kubeflow_pipelines_component_examples/filter_file_component/
```